### ISSUE TITLE
How to add a title to the figures when using the corrr package?

### ISSUE THREAD

**Issue by Levin928** [tidymodels/corrr#143]
### Tips for a helpful feature request: 

 * See our [contributing guidelines](https://github.com/tidymodels/tidymodels/blob/master/.github/CONTRIBUTING.md) and [development guide](https://www.tidymodels.org/contribute/) for more on design goals and how to contribute.

 * Please include a **minimal reproducible example,** a reprex, to demonstrate your feature idea when appropriate. If you've never heard of a reprex before, start by reading "[What is a reprex](https://github.com/tidyverse/reprex#what-is-a-reprex)" and following the advice there. A reproducible example can be effective at demonstrating the case for a new feature. 

 * We don't want you to use confidential data; you can use a dataset from [modeldata](https://modeldata.tidymodels.org/reference/index.html), blind the data, or simulate other data to demonstrate your bug. The functions [`caret::twoClassSim()`](https://www.rdocumentation.org/packages/caret/versions/6.0-84/topics/SLC14_1) or [`caret::SLC14_1()`](https://www.rdocumentation.org/packages/caret/versions/6.0-84/topics/SLC14_1) might be good tools to simulate data for you. 

 * Unless the feature is explicitly about parallel processing, please run sequentially. _Even if_ it is about parallel processing, please make sure that it runs sequentially first.

 * Use `set.seed()` to ensure any randomness in your code is reproducible.

 * Please check <https://stackoverflow.com/> or <https://community.rstudio.com/> to see if someone has already asked the same question (see: [Yihui's Rule](https://yihui.name/en/2017/08/so-gh-email/)). 

 * You might need to install these packages to create a reproducible example or share session info:

```r
install.packages(c("reprex", "sessioninfo"), repos = "http://cran.r-project.org")
```

When you are ready to file the feature ✨ request, please delete everything above this line:
< -- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->

## Feature

In situations when ...

**Comment by thisisdaryn** [comment]
The plots returned from `rplot()` and `network_plot()` are ggplot objects. You can use the `ggplot2::labs()`  or `ggplot2::ggtitle()` to add titles.

Example: `network_plot(correlate(mtcars)) + labs(title = "A network plot of mtcars variables")` 

(Similarly, you can use other functions such as `ggplot2::xlab()`, `ggplot2::ylab()` and theme functions of ggplot2 to modify the plots.)

**Comment by Levin928** [comment]
Thank you very much!



***@***.***
 
From: Daryn Ramsden
Date: 2021-07-18 04:22
To: tidymodels/corrr
CC: Levin928; Author
Subject: Re: [tidymodels/corrr] How to add a title to the figures when using the corrr package? (#143)
The plots returned from rplot() and network_plot() are ggplot objects. You can use the ggplot2::labs() or ggplot2::ggtitle() to add titles.
Example: network_plot(correlate(mtcars)) + labs(title = "A network plot of mtcars variables")
(Similarly, you can use other functions such as ggplot2::xlab(), ggplot2::ylab() and theme functions of ggplot2 to modify the plots.)

**Comment by juliasilge** [comment]
Let us know if you have further questions! 🙌

**Comment by Levin928** [comment]
Dear Julia, 

When using the R package corrr, I found another issue, maybe you can optimize it. As the bellow picture showed the legend is very close to the margin.  Thank you!

Best regards
Lijian




------------------------------------------------------------------
发件人：Julia Silge ***@***.***>
发送时间：2021年7月19日(星期一) 00:33
收件人：tidymodels/corrr ***@***.***>
抄　送：Levin928 ***@***.***>; Author ***@***.***>
主　题：Re: [tidymodels/corrr] How to add a title to the figures when using the corrr package? (#143)

Closed #143.

**Comment by juliasilge** [comment]
@Levin928 It looks like you tried to respond to this issue via email but there is no image attached; I recommend starting a new issue demonstrating your problem with a [reprex](https://www.tidyverse.org/help/).
